tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,144 @@
+
+The user can then chose to either
+
+1. Set a breakpoint at the op (equivalent to selecting in the node list).
+2. Continue to the op. This convenient option sets the breakpoint and then continues execution to the op.
+
+![Add a breakpoint via the graph explorer](images/context_menu.png)
+
+# Controlling Execution
+
+After selecting nodes, the user can continue execution for a certain number of session runs by clicking *Continue*:
+
+![The continue button](images/continue_button.png)
+
+Clicking the button opens a dialog that lets the user specify how many session runs to execute. Execution will pause at breakpoints (selected ops).
+
+![The continue dialog](images/continue_dialog.png)
+
+Via this dialog, the user may specify breakpoints based on conditions (in addition to the breakpoints that are based on selected ops):
+
+* When a tensor contains any bad (NaN, or +/- Infinity) values.
+* When a tensor contains any +/- Infinity values.
+* When a tensor contains any NaN values.
+* When the max value of a tensor exceeds some constant.
+* When the max value of a tensor is below some constant.
+* When the min value of a tensor exceeds some constant.
+* When the min value of a tensor is below some constant.
+* When the (max - min) value of a tensor exceeds some constant.
+* When the (max - min) value of a tensor is below some constant.
+* When the mean value of a tensor exceeds some constant.
+* When the mean value of a tensor is below some constant.
+* When the standard deviation of a tensor exceeds some constant.
+* When the standard deviation of a tensor is below some constant.
+
+These conditions bear much semblance to [filters of the TensorFlow debugger](https://www.tensorflow.org/programmers_guide/debugger#finding_nans_and_infs).
+
+When execution is paused, the next op can be stepped to via clicking *Step*. If a program runs multiple sessions, they will be listed within the Session Runs table under the node list.
+
+![Step to the next op](images/step.png)
+
+# Examining Runtime Graphs
+
+Runtime graphs for each device may be examined within the graph explorer.
+
+As noted above, users may right-click and select ops directly within the graph. When execution pauses at an op, the graph pans to it.
+
+![Exploring graphs](images/graph_explorer.png)
+
+# Examining the Values of Tensors
+
+When execution is paused, the values of output tensors for all selected ops are shown within the tensor values table (under the graph). The current op is shown in red.
+
+![Tensor value overview](images/tensor_value_overview.png)
+
+Also presented for each op are the name, count (the number of times the op has been executed), data type, and shape.
+
+Next to each op is a *health pill*, which visualizes the proportion of values within the tensor that fall under each of the six categories noted in the legend. A user might use health pills to for instance pinpoint ops that are culprits for producing undesired values (such as NaN).
```

Comment:
Also mention that hovering the mouse cursor above a health pill will open a tooltip which shows more information about the tensor, such as mean and standard deviation.

Reply:
Done.